app/src/main/cpp/ControllerDelegate.h

```diff
@@ -19,6 +19,8 @@ namespace crow {
 class ControllerDelegate;
 typedef std::shared_ptr<ControllerDelegate> ControllerDelegatePtr;
 
+enum class ControllerMode { None, DeviceMode, HandMode };
```

Comment:
Sounds good.

Reply:
At the end I removed the "Mode" suffix from the enum labels but left the enum name to `ControllerMode` since there is already a `Controller::type` which is a `DeviceType`.